pepper-apis/dsm-core/pom.xml

```diff
@@ -247,6 +247,14 @@
             <artifactId>gson</artifactId>
             <!-- <version>2.8.5</version> -->
         </dependency>
+        <dependency>
+            <groupId>com.brsanthu</groupId>
+            <artifactId>google-analytics-java</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.brsanthu</groupId>
+            <artifactId>google-analytics-java</artifactId>
+        </dependency>
```

Comment:
double or am I missing something?

Reply:
oh yes, I think intellij added it while importing too, will fix now